models/src/main/scala/coop/rchain/models/rholangn/Bindings.scala

```diff
@@ -0,0 +1,23 @@
+package coop.rchain.models.rholangn
+
+import coop.rchain.models._
+
+object Bindings {
+  def toProto(p: ParN): Par                             = BindingsToProto.toProto(p)
+  def toProto(ps: Seq[ParN]): Seq[Par]                  = ps.map(toProto)
+  def toProto(pOpt: Option[ParN]): Option[Par]          = pOpt.map(toProto)
```

Comment:
yes, this function can be removed, because I don't use it. But this function very useful to simplification conversion:
https://github.com/RhonixLabs/rhonix/blob/42c4813eda007098157a202f220fda29a380033d/models/src/main/scala/coop/rchain/models/rholangn/Bindings.scala#L9
Because I don't have to write `pOpt.map(BindingsToProto.toVar)` each time.